src/main/java/com/redhat/cloud/notifications/routers/BehaviorGroupMigrationService.java

```diff
@@ -67,6 +68,7 @@
         return session.withTransaction(tx -> {
             LOGGER.debug("[BG MIGRATION] Start");
             return Uni.createFrom().item(new AccountBehaviorGroupsAggregator())
+                    .onItem().call(ignored -> session.createQuery("DELETE FROM BehaviorGroup").executeUpdate())
```

Comment:
Unrelated to the issue mentioned above: we are now deleting all records from the `behavior_group` table before the migration starts. The delete is cascaded to the `behavior_group_action` and `event_type_behavior` tables which are both new. This does not trigger any cascade delete of existing production data.